addons/l10n_mx/models/chart_template.py

```diff
@@ -74,6 +74,6 @@ def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
             'company_id': company.id,
             'default_credit_account_id': account.id,
             'default_debit_account_id': account.id,
-            'show_on_dashboard': True,
+            'show_on_dashboard': False,
```

Comment:
Is this line intended in the diff? 
If yes, you should make a distinct commit/PR for that hunk.
If no, you should remove that from the PR